src/plugins/index_pattern_field_editor/public/components/flyout_panels/flyout_panels.scss

```diff
@@ -0,0 +1,33 @@
+.fieldEditor__flyoutPanels {
+  height: 100%;
+}
+
+.fieldEditor__flyoutPanel {
+  height: 100%;
+  overflow-y: auto;
+  padding: $euiSizeL;
+
+  &--lightestShade {
+    background-color: $euiColorLightestShade;
+  }
+  &--lightShade {
+    background-color: $euiColorLightShade;
+  }
+  &--mediumShade {
+    background-color: $euiColorMediumShade;
+  }
+  &--withFooter {
+    height: calc(100% - #{$euiSizeL * 3});
```

Comment:
This calculated height seems too static. What happens if they pass more than just a row of buttons? I'd recommend sticking with flex layouts and just grow the body content.